src/NServiceBus.ContainerTests/When_using_nested_containers.cs

```diff
@@ -190,19 +203,52 @@ public void Should_not_dispose_singletons_when_container_goes_out_of_scope()
             }
         }
 
-        class SingletonComponent : ISingletonComponent, IDisposable
+        [Test]
+        public void Should_dispose_all_IDisposable_components_in_child_container()
```

Comment:
The bane of my existence over the past week....

![blah](http://stream1.gifsoup.com/view7/2821067/fail-house-of-cards-o.gif)